packages/extension/src/routes/Step2.vue

```diff
@@ -204,4 +218,14 @@ export default {
     font-weight: 300;
   }
 }
+
+.btn-skip {
+  margin-top: 24px;
+  font-size: 11px;
+  font-weight: bold;
+  line-height: 16px;
+  letter-spacing: 1.6px;
+  color: var(--color-salt-90);
```

Comment:
We have the color palette defined here: https://github.com/dailynowco/daily-apps/blob/master/packages/components/src/styles/colors.pcss
So you don't need to hardcode set the color. In this case, you should even use the theme colors and not the actual colors themselves, as it should change color according to the user's theme preference.
https://github.com/dailynowco/daily-apps/blob/master/packages/components/src/styles/themes.pcss